pkg/scheduler/framework/session_plugins.go

```diff
@@ -232,11 +232,39 @@ func (ssn *Session) QueueOrderFn(lQ, rQ *queue_info.QueueInfo, lJob, rJob *podgr
 		}
 	}
 
-	// If no queue order funcs, order queue by CreationTimestamp first, then by UID.
-	if lQ.CreationTimestamp.Equal(&rQ.CreationTimestamp) {
-		return lQ.UID < rQ.UID
+	// Queue order not resolved, fallback to job CreationTimestamp first, then by UID.
+	lJ, rJ := ssn.getJobsForQueueOrder(lJob, rJob, lVictims, rVictims)
+	if lJ != nil && rJ != nil {
+		if !lJ.CreationTimestamp.Equal(&rJ.CreationTimestamp) {
+			lJ.CreationTimestamp.Before(&rJ.CreationTimestamp)
```

Comment:
Here you are calling `Before` but not using the return value, it happens again later.